resources/node.rb

```diff
@@ -44,12 +42,16 @@ def cluster_quorum_disk?(disk_name)
     powershell_out_with_options('(Get-ClusterQuorum).QuorumResource.Name').stdout.chomp == disk_name
   end
 
-  def install_windows_feature(feature)
-    to_array(feature).each do |f|
+  def cluster_quorum_fs_witness?
+    powershell_out_with_options('(Get-ClusterQuorum).QuorumResource.Name').stdout.chomp =~ /File Share Witness/
+  end
```

Comment:
This method should return if a specific file share witness is used (like `cluster_quorum_disk?` does) so that the value/path can be changed after first setup.

Reply:
This would be a nice to have imho

Reply:
Fixed